### PR TITLE
Provide non-sudo installation instructions in quickstarts

### PR DESCRIPTION
**Pull request by christiam**
This pull request refers quickstart readers to installation instructions via
PyPI and Bioconda:
Screen Shot 2022-03-02 at 5 07 12 PM

### CODE REVIEW

File: docs/source/quickstart-aws.rst [hunk 1]
```diff
@@ -43,17 +43,10 @@ In order to complete this quickstart, you will need to be familiar with the AWS
...
-.. code-block:: shell
-
-    sudo yum install -y python3-wheel
-    sudo pip3 install elastic-blast
-
-
-In some cases (e.g., not on the cloud), it may be preferable to run these commands without using "sudo" (which runs these commands with root permissions).  
-
-The instructions in this quickstart assume that you are working from the directory where you installed ElasticBLAST.
+Please follow the instructions on :ref:`tutorial_pypi` or
```
Comment (boratyng) [inline docs/source/quickstart-aws.rst:46]: I think that we should have the lines of code here on this page, rather than require users to follow a link.

Comment (tom6931) [reply docs/source/quickstart-aws.rst:46]: I agree.  For the quickstart, people should not have to navigate away to look things up.

Comment (christiam) [reply docs/source/quickstart-aws.rst]: Thanks for the feedback, this is done in b87e1a4.

File: docs/source/quickstart-gcp.rst [hunk 2]
```diff
@@ -49,20 +49,20 @@ As you work through this quickstart, you may occasionally see a message from the
...
+.. code-block:: bash
 
-    sudo apt-get install python-wheel
-    sudo pip3 install --upgrade setuptools
-    sudo pip3 install --upgrade pip
-    sudo pip3 install elastic-blast
+    [ -d .elb-venv ] && rm -fr .elb-venv
+    python3 -m venv .elb-venv
+    source .elb-venv/bin/activate
+    pip install elastic-blast=={VERSION}
```
Comment (boratyng) [inline docs/source/quickstart-gcp.rst:60]: Also in the interest if keeping things simple for a user, why not add `pip install wheel` here. Currently one must do it in GCP cloud shell, so every user will get an error right in the beginning. I am not sure if it is also needed in AWS cloud shell.

Comment (christiam) [reply docs/source/quickstart-gcp.rst:60]: Thanks, this is done in 2206fdb . I tested this in my GCP account's cloudshell and it worked well. I also tested this on my AWS account's cloudshell and `pip install wheel` was not needed.

**Review by boratyng — APPROVED** [review]

**Pull request merged**